boptimol/mean_potentials.py

```diff
@@ -0,0 +1,64 @@
+from gpytorch.means import ConstantMean, Mean
+import torch
+
+class MorseMean(Mean):
+    
+    """ A configurable mean function using the Morse potential
+
+    Uses BOTorch to pick the next sample using Expected Improvement
```

Comment:
Might be worth indicating the formula in the docs, since not everyone knows this.